### DISCUSSION TITLE
Option to label within a list

### DISCUSSION THREAD

**Discussion by bhupesh-sf** [community/community#72617] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Sometimes a list gets bigger so need a way to label repos within the list so I can search and group them together. For e.g. I have create a list for ML. Now in this list many are frameworks to create ml models, some are for deployment, some are to learn ML and many are just collection of open source models. Collectively it has become a big list. 

I really wish now to quickly sort/search them so I can go to my desired topic and explore that repo.

**Comment by blackdoc111** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Sometimes a list gets bigger so need a way to label repos within the list so I can search and group them together. For e.g. I have create a list for ML. Now in this list many are frameworks to create ml models, some are for deployment, some are to learn ML and many are just collection of open source models. Collectively it has become a big list. 
> 
> I really wish now to quickly sort/search them so I can go to my desired topic and explore that repo.

**Comment by blackdoc111** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Sometimes a list gets bigger so need a way to label repos within the list so I can search and group them together. For e.g. I have create a list for ML. Now in this list many are frameworks to create ml models, some are for deployment, some are to learn ML and many are just collection of open source models. Collectively it has become a big list. 
> 
> I really wish now to quickly sort/search them so I can go to my desired topic and explore that repo.

**Reply by KrMustaf** [reply]
Comment ça marche
